test/Frontend/OpenQASM3/bitops.qasm

```diff
@@ -107,3 +107,5 @@ f = e | d;
 // MLIR: [[NOT_CBIT:%.*]] = "oq3.cast"([[NOT]]) : (i1) -> !quir.cbit<1>
 // MLIR: oq3.variable_assign @f : !quir.cbit<1> = [[NOT_CBIT]]
 f = !f;
+// equivalent to "!"
+f = ~f;
```

Comment:
Could you please add some mlir checks for the output on the lines below?

See above as an example
```
// MLIR: [[NOT_CBIT:%.*]] = "oq3.cast"([[NOT]]) : (i1) -> !quir.cbit<1>
// MLIR: oq3.variable_assign @f : !quir.cbit<1> = [[NOT_CBIT]]
```

These are verified by FileCheck](https://llvm.org/docs/CommandGuide/FileCheck.html).

Reply:
NP. Thanks

Reply:
Thanks for helping me understand this.

I just checked `./qss-compiler ../../test/Frontend/OpenQASM3/bitops.qasm | FileCheck ../../test/Frontend/OpenQASM3/bitops.qasm --check-prefix MLIR` runs correctly without error.

Reply:
Thanks, this looks great!